src/Objects/Boy.java

```diff
@@ -0,0 +1,51 @@
+package Objects;
+import Place.*;
+import Enums.Status;
+import Interfaces.Playable;
+
+public class Boy extends Person implements Playable {
+    private Status stat;
+
+    public Boy(String name, Status type, int force, int height, Place place) {
+        super(name,type, force, height, place);
+        this.stat = type;
+    }
+    public String describe1() {
+        return this.getName() + " " + this.stat.getStat();
+    }
+    public String changeStat(Person p) {
+        point = (int) (point + 10 * Math.random());
+        if (point > 2) {
+            this.stat = Status.THINKING;
+            return this.describe1();
+        } else {
+            this.stat = Status.NORMAL;
+            return this.describe1();
+        }
+    }
+    int point = 0;
+
+    @Override
+    public String play(int point, Status stat, Game g) {
+        point = (int) (Math.random() * 20);
+        if (point > 15) {
+            this.stat = Status.THINKING;
+            return "Thinking only because of the "  + g.getType().getStat()+ " " + g.getName();
```

Comment:
Как и в методе выше `return "Thinking not only because of the " + g.getType().getStat()+ " " + g.getName();` можно вынести после if else.

И если что-то нужно вывести в консоль. то лучше, думаю, будет System.out.println() вставлять сразу в метод. Чтобы он не возвращал String. Иначе может быть странно, что метод play или changeStat возвращают еще какую-то строку, хотя из названия так не скажешь